Automattic/jetpack
# My Plan: Add Offer Reset project new plans (Jetpack Security, Jetpack Complete)

#### Changes proposed in this Pull Request:

* Configure constants for the new plans that are part of the Offer Reset project: Jetpack Security Daily, Jetpack Security Real-Time, and Jetpack Complete.
* Enable these plans in the My Plan page.
* Add a SVG icon for each of them.

#### Jetpack product discussion

* Ticket: 1169247016322522-as-1187323761633593

#### Does this pull request change what data or activity we track or use?

* No. It doesn't.

#### Testing instructions:

Notes: 

* Since we haven't configure yet these plans in our backend, it isn't possible to test this unless you have access to the code (*). It might be possible to see the new products by modifying the state of the app (I haven't tried it yet).
* Please, don't forget to review the copy associated with each plan (the icon too).

(*) If you want to edit the code to test each plan, you can modify the `/jetpack/_inc/client/my-plan/index.jsx` and add to the start of the `MyPlan` component the following:
```
export function MyPlan( props ) {
    // Set the slug of the plan you want to check
	if ( props.sitePlan ) {
		props.sitePlan.product_slug = 'jetpack_security_daily';
	}
    // the rest of the code goes here...
```

Instructions:

* Run this PR (locally or with the Beta Tester plugin).
* Buy one of the new plans.
* Visit the My Plan page.
* Check that you see your new plan in the My Plan section.

##### Jetpack Security Daily
<img src="https://user-images.githubusercontent.com/3418513/89464885-9030f980-d747-11ea-8c1b-f90260b834a3.png" width="550" alt="Jetpack Security Daily">

##### Jetpack Security Real-Time
<img src="https://user-images.githubusercontent.com/3418513/89464911-9a52f800-d747-11ea-9e57-e565ffad0c59.png" width="550" alt="Jetpack Security Real-Time">

##### Jetpack Complete
<img src="https://user-images.githubusercontent.com/3418513/89464798-6a0b5980-d747-11ea-8175-b2aa13ddc76d.png" width="550" alt="Jetpack Complete">

#### Proposed changelog entry for your changes:

* Configure Jetpack Security Daily, Jetpack Security Real-Time, and Jetpack Complete as part of the Offer Reset project.
